tests/Unit/ApplicationWatchPathsTest.php

```
@@ -0,0 +1,165 @@
+
+class ApplicationWatchPathsTest extends TestCase
+{
+    public function test_is_watch_paths_triggered_returns_false_when_watch_paths_is_null()
+    {
+        $application = new Application();
+        $application->watch_paths = null;
+
+        $modified_files = collect(['docker-compose.yml', 'README.md']);
+
+        $this->assertFalse($application->isWatchPathsTriggered($modified_files));
+    }
```

PHPMD (2.15.0)

The method test_is_watch_paths_triggered_returns_false_when_watch_paths_is_null is not named in camelCase. (undefined)

(CamelCaseMethodName)